iocage_lib/ioc_common.py

```diff
@@ -770,3 +770,12 @@ def match_to_dir(iocroot, uuid, old_uuid=None):
         return matches[0]
     else:
         return None
+
+
```

Comment:
The existence of that rings all sort of bells.

That means IOExec is no longer contained. Imagine you're using IOExec and somewhere along the way of reading stdout/stderr some exception occurs that you're not expecting and the generators are not consumed. You have not finished reading stderr/stdout, the buffer gets full and the process never dies, waiting the pipes to be read.

I believe the way to solve it is making IOExec a context manager.